src/components/interaction-designer/block-editors/GroupMembershipEditor.vue

```diff
@@ -175,19 +180,20 @@ export class GroupMembershipEditor extends mixins(Lang) {
     this.$emit('group-search', e)
   }
 
-  onGroupAdd(name: string): void {
+  async onGroupAdd(name: string): Promise<void> {
     const newGroup: IGroupMembership = {
       group_key: name,
       group_name: name,
     }
 
     this.userAddedGroups.push(newGroup)
```

Comment:
I have kept `userAddedGroups` local which makes adding groups undoable. Hope that is okay.